### ISSUE TITLE
MissingArguments on use function

### ISSUE THREAD

**Issue by szepeviktor** [squizlabs/PHP_CodeSniffer#2574]
PHPCS sees `use function` as actual function usage.

```php
declare(strict_types=1);

namespace WordPressModel\Factory;

use DateTimeZone;
// phpcs:ignore WordPress.PHP.StrictInArray.MissingArguments
use function in_array;
use function is_numeric;
```

please advise

**Comment by szepeviktor** [comment]
Could it be a WPCS-specific issue??

**Comment by jrfnl** [comment]
@szepeviktor It is, or rather it was. This was fixed in WPCS 1.0.0 which was released quite a while back. Upgrading should help ;-)

**Comment by szepeviktor** [comment]
Thank you @jrfnl !
